hub/powertoys/awake.md

```diff
@@ -68,3 +69,8 @@ When setting the value for the `--time-limit` parameter, both of these formats w
 
 - PowerToys.Awake.exe --time-limit 36000
 - PowerToys.Awake.exe --time-limit=36000
+
+When setting the value for the `--expire-at` parameter, these formats will be accepted based on [DateTime.Parse](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.parse)
+
+- PowerToys.Awake.exe --expire-at=17:00:00 will expire at 5PM of the current day
```

Comment:
This is more of a general comment, but when we document commands, we need to make sure that we wrap them in single quotes, to separate from the explanatory text. Right now it all blends together.